src/System Application/App/Extension Management/src/ExtensionManagement.Page.al

```diff
@@ -273,6 +260,75 @@ page 2500 "Extension Management"
                     ToolTip = 'Delete the data of orphaned extensions.';
                 }
             }
+
+            group("Develop in VS Code")
+            {
+                Caption = 'Develop in VS Code';
+                ToolTip = 'Set up your local of actions ';
+
+                action("Open Source in VS Code")
+                {
+                    ApplicationArea = All;
```

Comment:
Please remove all occurrences of `ApplicationArea = All,` as it was already defined at object level.